### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by vietnogi**
ping @indytechcook, @tladendo

### CODE REVIEW

**Comment by tladendo** [comment]
The code looks good. Still wanna test, but there's one thing I want to ask about: `client/app/core/styles` vs `client/styles`? Thoughts? I'm in favor of the latter for global styles, since some other global stuff will live in `client`.

**Comment by vietnogi** [comment]
@tladendo, I chose core because we place our core javascript files there

**Comment by indytechcook** [comment]
lgtm, i'm sure it will break some stuff but we can iterate fast :)

**Comment by indytechcook** [comment]
opps, nm.  build is broken.  Per IM conversatoin I'll push this change to gulp:

```javascript
gulp
   .src('./src/client/app/**/*.scss')
   .pipe($.compass({
     css: '.tmp/',
     sass: 'src/client/app/'
   }))
   .pipe(gulp.dest('.tmp/'));
```javascript

**Comment by indytechcook** [comment]
SCSS compiles but it doesn't appear that any of the javascript is injected.

**Comment by indytechcook** [comment]
Just running gulp scss which only does the jade and scss works fine.  The issue is with the auto injection of the javascript which I am almost done removing in another branch.  I say we merge this one, then I push my fix for the javascript in the other branch.

Sound good?

File: gulpfile.js [hunk 1]
```diff
@@ -63,14 +63,16 @@ gulp.task('scss', ['clean-styles', 'jade'], function () {
   log('Compiling SCSS --> CSS');
 
   return gulp
-    .src(config.scss)
-    .pipe(gulp.dest(config.scssBuild))
-    .pipe($.compass(config.compass))
-    .pipe(gulp.dest(config.temp));
+    .src('./src/client/app/**/*.scss')
+    .pipe($.compass({
+      css: '.tmp/',
```
Comment (tladendo) [inline gulpfile.js]: This should use config variables. This will break, seeing as we've changed `.tmp` to `tmp` in another commit.

Comment (indytechcook) [inline gulpfile.js]: Good call,

**Comment by vietnogi** [comment]
@indytechcook , jade file donst appear to be compiling when running dev serve. Currently I would have to save index.jade manually to kick off the jade task.

**Comment by indytechcook** [comment]
Thanks @vietnogi i have some build fixes in another branch.  as soon as this is merged and I'll pull this in and get them working

**Pull request merged**